core/domain/exp_domain.py

```diff
@@ -5168,7 +5168,7 @@ def update_states_from_model(
     # incompatible changes are made to the exploration schema in the YAML
     # definitions, this version number must be changed and a migration process
     # put in place.
-    CURRENT_EXP_SCHEMA_VERSION = 60
+    CURRENT_EXP_SCHEMA_VERSION = 61
```

Comment:
YOu also need to increase the `CURRENT_STATE_SCHEMA_VERSION` in feconf and also add one more function, please look at the previous migrations to see what I mean.